### PR TITLE
fix(typeahead): Update css for typeahead dropdown

### PR DESCRIPTION
**Pull request by SMahil**
Update css for typeahead dropdown
screen shot 2017-04-22 at 1 10 54 pm
screen shot 2017-04-22 at 1 11 07 pm

### CODE REVIEW

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1731-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss [hunk 1]
```diff
@@ -2,106 +2,69 @@
...
+    position: absolute;
+    top: 100%;
+    left: 0;
+    z-index: 1000;
+    float: left;
+    min-width: 160px;
+    padding: 5px 0;
+    margin: 2px 0 0;
+    list-style: none;
+    font-size: 12px;
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss]: Should be the default 14px so as not to cause any "jumping".

File: src/assets/stylesheets/shared/_custom-mixins.scss [hunk 2]
```diff
@@ -90,31 +90,49 @@
     -moz-border-style: $borderStyle;
 }
 
-//multiline truncate
-@mixin multiline-truncate($line-height, $line-count, $bg-color) {
+@mixin background($args...) {
+  background: #ffffff; /* Old browsers */ /* FF3.6-15 */ /* Chrome10-25,Safari5.1-6 */
```
Comment (mindreeper2420) [inline src/assets/stylesheets/shared/_custom-mixins.scss]: Set to $color-pf-white

File: src/assets/stylesheets/shared/_custom-mixins.scss [hunk 3]
```diff
@@ -90,31 +90,49 @@
...
+      content: "\02026";
+      float: right;
+      position: relative;
+      top: -$lineheight;
+      left: 100%;
+      width: $ellipsiswidth;
+      margin-left: -$ellipsiswidth;
+      padding-right: 5px;
+      text-align: right;
+      background: #fff;
```
Comment (mindreeper2420) [inline src/assets/stylesheets/shared/_custom-mixins.scss]: $color-pf-white

File: src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss [hunk 4]
```diff
@@ -2,106 +2,69 @@
...
+    -webkit-box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+    box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+    background-clip: padding-box;
+    margin-top: -1px;
+    width: 100%;
+    li {
+      a{
+        border-color: transparent;
+        border-style: solid;
+        border-width: 1px 0;
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss]: What is the second number for?

File: src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss [hunk 5]
```diff
@@ -2,106 +2,69 @@
...
+    color: $color-pf-blue;
+  }
+  .details-dropdown {
+    /* width: 100%; */
+  }
+  .item-ul{
+    position: absolute;
+    top: 100%;
+    left: 0;
+    z-index: 1000;
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss]: Will this cause conflicts? Can the default z-index be used from the dropdown component?

File: src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss [hunk 6]
```diff
@@ -2,106 +2,69 @@
...
+      a{
+        border-color: transparent;
+        border-style: solid;
+        border-width: 1px 0;
+        padding: 1px 10px;
+        display: block;
+        clear: both;
+        font-weight: 400;
+        color: $color-pf-black-800;
+        line-height: 1.66666667;
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss]: You can simplify to 1.5

File: src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss (hunk 5 above)
Comment (SMahil) [inline src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss]: I am using this css from dropdown-menu class but without display

File: src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss (hunk 6 above)
Comment (SMahil) [inline src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss]: this is from patternfly

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1731-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1731-fabric8-planner.int.rdu2c.fabric8.io

**Comment by mindreeper2420** [comment]
@SMahil what is the Search button for? Is searching allowed for Areas and Iterations now?

**Comment by SMahil** [comment]
@mindreeper2420 Need to ask Micheal. He added funtionality

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1731-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1731-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.html [hunk 7]
```diff
@@ -1,26 +1,29 @@
...
+    <div *ngIf="searchValue" class="dropdown-search col-md-6 padding-0">
+      <input type="hidden" name="" value="">
+      <div class="input-group">
+        <input id="valueSearchInput" #valueSearch
+          (keyup)="filterValue($event)" autocomplete="off"
+          class="combobox form-control" />
+        <ul class="item-ul dropdown-list" #valueList>
+          <li *ngIf="!values.length" class="textCenter"><div class="spinner spinner-sm"></div></li>
+          <li *ngIf="!filteredValues.length">No matches found.</li>
+          <li class="item-li"*ngFor="let item of filteredValues;" [attr.data-value]="item.key" (click)="showValueOnInput(item)" [id]="item.key">
```
Comment (nimishamukherjee) [inline src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.html]: Padding needs to be added before No matches found text:

nomatches

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1731-fabric8-planner.int.rdu2c.fabric8.io

**Comment by SMahil** [comment]
@nimishamukherjee changes are done. Please confirm

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1731-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1731-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1731-fabric8-planner.int.rdu2c.fabric8.io

**Pull request merged**